manageiq-gems-pending.gemspec

```diff
@@ -29,6 +29,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency "nokogiri",                "~> 1.14", ">= 1.14.3"
   s.add_runtime_dependency "sys-proctable",           "~> 1.2.5"
   s.add_runtime_dependency "sys-uname",               "~> 1.2.1"
+  s.add_runtime_dependency "win32ole",                "~> 1.8.8" # this gem was extracted in ruby 3 - required if we use wmi on windows
```

Comment:
I remember we used to use this for smartproxy on windows

Do we still need this?

Reply:
I can't fine `WmiMswin` used anywhere in our code base

Reply:
ok, `vm.rb` uses wmi gook.
think there is some :fire: opportunity here

Reply:
yeah, I think so too... it's a definite squirrel though